Mathlib/Order/RelIso/Basic.lean

```diff
@@ -575,16 +588,31 @@ theorem toEquiv_injective : Injective (toEquiv : r ≃r s → α ≃ β)
 instance : CoeOut (r ≃r s) (r ↪r s) :=
   ⟨toRelEmbedding⟩
 
--- see Note [function coercion]
-instance : CoeFun (r ≃r s) fun _ => α → β :=
-  ⟨fun f => f⟩
```

Comment:
Probably should be `porting note: removed`

Reply:
Okay, I tried and failed to fix this from my phone. I'll have a proper look on Monday.

Reply:
I've fixed this note now